### PR TITLE
Handle URL change/suppression on loading  

### PR DESCRIPTION
**Pull request by mc2**
See https://webarchive.jira.com/browse/WEBDEV-3158

WEBDEV-3158

### CODE REVIEW

**Comment by mc2** [comment]
@cdrini 
We had a day of checking and speccing this.  Please see https://internetarchive.slack.com/archives/G6TNCRTNK/p1582842343006400
Need to rewrite this request.

**Comment by mc2** [comment]
Ready to be reviewed. Thanks!

File: tests/e2e/base-ia.test.js [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+  await t.click(flipRight);
+  await t.expect(hash()).contains('#page/n');
+  await t.expect(hash()).contains('/mode/2up');
+  await t.click(flipLeft);
+  await t.expect(hash()).contains('#page/n');
+  await t.expect(hash()).contains('/mode/2up');
+  // Cookie set by page navigation creates hash on navigate to canonical URL
+  await t.navigateTo('demo-ia-plato.html');
+  await t.expect(hash()).contains('#page/n');
+  await t.expect(hash()).contains('/mode/2up');
```
Comment (nsharma123) [inline tests/e2e/base-ia.test.js]: Can we test something else here when you click on any selector? e.g. click on 1up or thumbs view and then see if respected view mode is appearing on URL. Thanks!

Comment (mc2) [reply tests/e2e/base-ia.test.js]: Thanks for the suggestion. All kinds of tests can be added. I copied the existing base test, fixing it for the mobile menu. I only added tests for this specific Jira/issue, code changes.

There will likely be more code changes and we should add more tests for that code.

File: tests/e2e/base-ia.test.js [hunk 2]
```diff
@@ -0,0 +1,139 @@
+/* global fixture */
```
[the remark this answers is not in the document]
Comment (mc2) [reply tests/e2e/base-ia.test.js]: Yes, in both the plugins and especially the zoom buttons in the mobile menu plugin

[the remark this answers is not in the document]
Comment (mc2) [reply tests/e2e/base-ia.test.js]: And thanks for testing. I'll look at the red X above

[the remark this answers is not in the document]
Comment (cdrini) [reply tests/e2e/base-ia.test.js]: Or, is there a difference in the _test_ file?

[the remark this answers is not in the document]
Comment (cdrini) [reply tests/e2e/base-ia.test.js]: (Feel free to use https://58-review-br-es6ify-b09jkq.archive.org/ for more testing :+1: Jumping on it + yarn installing will pull from this branch)

**Comment by mc2** [comment]
Updated to es6 in new branch

**Pull request closed without merging**